### DISCUSSION TITLE
when I put an image and the background of the application is another image

### DISCUSSION THREAD

**Discussion by AngelCard** [TomSchimansky/CustomTkinter#1887] [Q&A]
Hello, I am having a problem with my application. When I put an image and the background of the application is another image, when I run it, the second image appears with a white background. It should be noted that the second image is .PNG

`import customtkinter as ctk
from PIL import Image

#Default Mode
ctk.set_appearance_mode('dark')
ctk.set_default_color_theme('green')

#Crear Ventana
root = ctk.CTk()

root.title('EduConter - Login')
root.after(0, lambda:root.state('zoomed'))

#Fondo Login
fondoLogin = ctk.CTkImage(
    light_image=Image.open('fondoLogin.jpg'),
    dark_image=Image.open('fondoLogin.jpg'),
    size=(root.winfo_screenwidth(),root.winfo_screenheight()))
labelFondoLogin = ctk.CTkLabel(master=root, image=fondoLogin, text='')
labelFondoLogin.place(relx=0.5, rely=0.5, anchor='center')

#Logo
logo = ctk.CTkImage(
    light_image=Image.open('logoEduConter.png'),
    dark_image=Image.open('logoEduConter.png'),
    size=(346,243))
imagen_tk = Image
labelFondoLogin = ctk.CTkLabel(master=root, image=logo, text='')
labelFondoLogin.place(relx=0.5, rely=0.5, anchor='center')

root.mainloop()`

**Comment by avalon60** [comment]
Maybe this thread will help:
https://github.com/TomSchimansky/CustomTkinter/discussions/1643

**Reply by AngelCard** [reply]
unfortunately it didn't help me, I need to know how to change the background of a label to transparent

**Reply by Adarsh-365** [reply]
i think the problem is with your image  even if image are .png they might have white background use https://www.remove.bg/upload site to remove the background of image and try again

**Reply by avalon60** [reply]
Yes, you can use GIMP or other image manipulation apps to make a transparent background.
